src/helper_functions.jl

```diff
@@ -0,0 +1,12 @@
+export get_dist_moment_ind
+export get_dist_moments_ind_range
+
+function get_dist_moment_ind(NProgMoms::Vector{Int}, i::Int, m::Int)
```

Comment:
Can you add some docstrings in this file to describe the arguments (i, m) for how this function is used?